aten/src/ATen/native/QuantizedLinear.cpp

```diff
@@ -276,8 +275,18 @@ Tensor fbgemm_pack_quantized_matrix(
   return cpp_custom_type_hack::create(std::move(ptr), weight.options());
 }
 
-float raw_uint16_to_fp16(unsigned short value)
-{
+Tensor fbgemm_pack_quantized_matrix(
+    const Tensor& weight,
+    int64_t K,
+    int64_t N) {
+  TORCH_WARN(
+      "fbgemm_pack_quantized_matrix(weight, K, N) will be deprecated soon."
```

Comment:
we don't even need a warning imho - it's not really a public API and if some serialized models use it we don't want to spam the output.

Reply:
The warning gives us a good way to spot-check if LATTE still has this stuff running on servers, so I think we should keep it

Also, this is technically a public API since it doesn't have a leading underscore